docs/upgrade.html

```diff
@@ -39,7 +39,8 @@ <h5><a id="upgrade_250_notable" href="#upgrade_250_notable">Notable changes in 2
         <a href="https://github.com/apache/kafka/tree/2.5/examples">examples</a> folder. Check out
         <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-447%3A+Producer+scalability+for+exactly+once+semantics">KIP-447</a>
         for the full details.</li>
-    <li>Deprecated <code>KafkaStreams.store(String, QueryableStoreType)</code> and replaced it with <code>KafkaStreams.store(StoreQueryParameters)</code>.</li>
+    <li>Provided support to query stale stores(for high availability) and the stores belonging to a specific partition by deprecating <code>KafkaStreams.store(String, QueryableStoreType)</code> and replacing it with <code>KafkaStreams.store(StoreQueryParameters)</code>.</li>
```

Comment:
break this up into two separate updates so that we also cite the kafkaStreams.keyQueryMetadata() api?

Reply:
Makes sense, that is also an important api.

Reply:
Do we need to break up this line to add kafkaStreams.keyQueryMetadata()? That is a separate endpoint that returns key specific information, i.e. partition and hosts containing stale tasks in addition to the active hosts which we were able to fetch earlier.

Reply:
Something like ` <li>Added a new public api <code>KafkaStreams.queryMetadataForKey(String, K, Serializer) to get detailed information on the key being requested. It provides information about the partition number where the key resides in addition to hosts containing the active and standby partitions for the key.</code></li>
  `

Reply:
@vinothchandar can you review this again.

Reply:
Thanks @abbccdda . Addressed both the comments in the latest commit.